pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -243,6 +243,7 @@ public void deleteSegmentFile() {
 
   // Segment end criteria
   private volatile long _consumeEndTime = 0;
+  private volatile boolean _hasMessagesFetched = false;
```

Comment:
nit: `hasFetchedAnyMessages` or  just`hasFetchedMessages`?

Reply:
I prefer the current name to be consistent with other variables, e.g. `_numRowsConsumed`, `_forceCommitMessageReceived` etc.